app/views/forumthreads/show.html.erb

```diff
@@ -1,6 +1,12 @@
 <%= link_to @thread.forum.group, forumgroup_path(@thread.forum.group) %> → <%= link_to @thread.forum, @thread.forum %> → <%=truncate(@thread.title, length: 60, omission: " …") %>
-<h1><%= render partial: "labels/label", locals: {label: @thread.label} %><%= title @thread.title %></h1>
-
+<h1>
+  <%= render partial: "labels/label", locals: {label: @thread.label} %><%= title @thread.title %>
+  <% if params[:reverse] %>
+    <%= link_to "Reverse Replies", @thread, class: "btn right blue" %>
+  <% else %>
+    <%= link_to "Reverse Replies", forumthread_path(@thread, reverse: true), class: "btn right blue" %>
+  <% end %>
```

Comment:
This would not work. `params[:reverse]` is a string, and not a boolean, so you can't just get the opposite using `!params[:reverse]`. I can however, do `reverse = true if params[:reverse] == "true"` and then pass `!reverse` into the `link_to`.